code/modules/mob/living/carbon/human/life.dm

```diff
@@ -549,7 +554,10 @@
 		failed_last_breath = 1
 	else
 		failed_last_breath = 0
-		adjustOxyLoss(-5)
+		if(disabilities & ASTHMA)
+			adjustOxyLoss(rand(-5,0))

```

Comment:
An asthmatic takes less oxygen damage while failing to breathe? Wut?

Reply:
What indeed? This is if they pass their last_breath, hence `failed_last_breath = 0`